http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfigurationBuilder.java

```diff
@@ -195,6 +196,31 @@ protected OidcClientConfiguration internalBuild(final OidcJsonConfiguration oidc
 
         oidcClientConfiguration.setTokenSignatureAlgorithm(oidcJsonConfiguration.getTokenSignatureAlgorithm());
 
+        String tmpLogoutPath = oidcJsonConfiguration.getLogoutPath();
+        log.debugf("property LOGOUT_PATH: " + tmpLogoutPath);
```

Comment:
The references to "property" in the debug statements should be removed.

Reply:
done